### ISSUE TITLE
Ensure that icons meanings are clear with users

### ISSUE THREAD

**Issue by apferrar** [pulibrary/pulfalight#126]
If icons are retained, make sure that they refer to the same thing consistently and do not conflate intellectual description with physical arrangement. (i.e. a collection is not a box; a file is not necessarily a folder). Additionally, the icon that indicates "repository" should be changed to "collection" as it refers to collection information, not the repository (i.e. https://pulfalight-staging.princeton.edu/?q=&search_field=all_fields&view=compact )

**Comment by jrgriffiniii** [comment]
While it is possible to support this with LUX, but popovers don't work on many mobile devices. Further, these would not be useful for screen readers.

However, they do provide some value to users.

**Comment by jrgriffiniii** [comment]
Please see the following icons for an example:

![image](https://user-images.githubusercontent.com/1443986/77357431-8a012a00-6d1e-11ea-8eb0-0d1613dbdb23.png)

**Comment by jrgriffiniii** [comment]
These icons are valuable when providing instruction to users on how to properly navigate collections in PULFA.

**Comment by jrgriffiniii** [comment]
Are we in the position where we might be able to update the icons themselves?

**Comment by jrgriffiniii** [comment]
This should include an evaluation of what ArcLight is doing to select these icons.

**Comment by sdellis** [comment]
@jrgriffiniii This could be as simple as using the `title` attribute on the icon. I can verify that I tested and this will work in all modern browsers now.

**Comment by jrgriffiniii** [comment]
This will not require wireframing then.

**Comment by tpendragon** [comment]
Added to our usability testing ticket. Closing this.
